src/bots/proof-of-humanity/change-state-to-pending.js

```diff
@@ -0,0 +1,68 @@
+const { gql } = require('graphql-request')
+
+// changeStateToPending(address _submissionID, address[] calldata _vouches)
+// require(submission.status == Status.Vouching, "Wrong status");
+// require(round.hasPaid[uint(Party.Requester)], "Requester didn't pay his fees");
+// require(request.vouches.length >= requiredNumberOfVouches, "Not enough valid vouches");
```

Comment:
Can these comments be removed?

Reply:
> Is there a reason you have commented out require statements throughout?

I think the idea was to document when this will run.

Reply:
Let's put them in a more readable format.

Reply:
Yeah, I explained it in the description and gave an example:

>I have structured the code into one module per action callback to clarify the relationship between the queries and the contract's required conditions.

What format do you suggest?

The idea is that this makes it super easy to copy and search in the contract where the logic is.

Reply:
I think the part that makes it hard to read is the error messages. As the error message are the negation of the statement when I read a line I have to do a little mental gymnastics each time to figure out exactly what it is saying. And overall reading through code is more difficult than a `docstring` should be. It would be much easier to follow if the conditions are in plain english. You can leave the function signature at the top if you want it to be easily searchable in the smart contract.

e.g.
`
changeStateToPending(address _submissionID, address[] calldata _vouches)
Change the state of a submission to pending. Submissions must have a status of Vouching, have paid the fees, and have at least the required number of vouches. 
`